istioctl/cmd/root.go

```diff
@@ -199,36 +199,36 @@ debug and diagnose their Istio mesh.
 	experimentalCmd.AddCommand(precheck.Cmd(ctx))
 	experimentalCmd.AddCommand(proxyconfig.StatsConfigCmd(ctx))
 	experimentalCmd.AddCommand(checkinject.Cmd(ctx))
-	rootCmd.AddCommand(waypoint.Cmd(ctx))
-	rootCmd.AddCommand(ztunnelconfig.ZtunnelConfig(ctx))
+	rootCmd.AddCommand(setNotSupportedCmd(waypoint.Cmd(ctx).Name()))
+	rootCmd.AddCommand(setNotSupportedCmd(ztunnelconfig.ZtunnelConfig(ctx).Name()))
```

Comment:
We can support these commands when Ambient support is added in Sail Operator as a TP or GA feature. Can you add a TODO here as follows.

TODO: revisit once ambient mode is supported in Sail Operator.

Reply:
Done